common/test/acceptance/tests/lms/test_lms_course_home.py

```diff
@@ -164,3 +167,77 @@ def test_course_search_a11y(self):
             ]
         })
         course_search_results_page.a11y_audit.check_for_accessibility_errors()
+
+
+class CourseOutlineTest(UniqueCourseTest):
+    """
+    Test Suite to verify the course outline page on the LMS.
+    """
+    USERNAME = "STUDENT_TESTER"
+    EMAIL = "student101@example.com"
+
+    def setUp(self):
+        """
+        Initialize pages and install a course fixture.
+        """
+        super(CourseOutlineTest, self).setUp()
+
+        self.course_home_page = CourseHomePage(self.browser, self.course_id)
+        # Install a course with sections and problems
+        self.course_fix = CourseFixture(
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run'],
+            self.course_info['display_name'],
+            start_date=datetime.now() + timedelta(days=-10),
+            end_date=datetime.now() + timedelta(days=10)
+        )
+
+        self.course_fix.add_children(
+            XBlockFixtureDesc('chapter', 'Test Section').add_children(
+                XBlockFixtureDesc('sequential', 'Test Subsection', metadata={
+                    'due': (datetime.now() + timedelta(days=-20)).isoformat(),
+                    'format': 'Homework'
+                }).add_children(
+                    XBlockFixtureDesc('problem', 'Test Problem', data=load_data_str('multiple_choice.xml')),
+                )
+            ),
+        ).install()
+        # Auto-auth register for the course.
+        auto_auth(self.browser, self.USERNAME, self.EMAIL, False, self.course_id)
+
+    def change_course_pacing_to_self_paced(self):
+        """
+        Change the course pacing from Instructor Paced to Self-paced course
+        for a live course.
+        """
+        # Change start date to future to be able to change pacing
```

Comment:
You don't need 1 comment to explain your next line. Make sure your doc string describes the intention of this method and you don't have to comment for each line.